### PR TITLE
Add guidance semantics and sequence

### PR DESCRIPTION
**Pull request by ferBonnin**
#### Description of changes

Added guidance and info& examples to Semantics and sequence

#### Pull request checklist

- [ ] Addresses an existing issue: Fixes #0000
- [ ] Added relevant unit test for your changes. (`yarn test`)
- [ ] Verified code coverage for the changes made. Check coverage report at: `<rootDir>/test-results/unit/coverage`
- [ x] Ran precheckin (`yarn precheckin`)
- [ x] (UI changes only) Added screenshots/GIFs to description above
- [x ] (UI changes only) Verified usability with NVDA/JAWS

info & examples CSS content

Do and Dont's Semantics

### CODE REVIEW

File: src/content/test/semantics/data-tables.tsx [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+            (Applying <Markup.Code>role="presentation"</Markup.Code> is appropriate for layout tables, because their table semantics do not
+            communicate any meaningful relationships.)
+        </p>
+
+        <h2>How to fix</h2>
+        <p>
+            Do not use <Markup.Code>role="presentation"</Markup.Code> on any of the semantic elements in a data table:
+            <ul>
+                <li>{'<table>'}</li>
+                <li> {'<tr>'}</li>
```
Comment (peterdur) [inline src/content/test/semantics/data-tables.tsx]: There are extra spaces between `<li>` and opening braces `{` here and on following lines - but they do not cause any problems.
```suggestion
                <li>{'<tr>'}</li>
```

File: src/content/test/semantics/letter-spacing.tsx [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+        <h3>WCAG success criteria</h3>
+        <Markup.Links>
+            <Markup.HyperLink href="https://www.w3.org/WAI/WCAG21/Understanding/info-and-relationships.html">
+                Understanding Success Criterion 1.3.1: Info and Relationships
+            </Markup.HyperLink>
+            <Markup.HyperLink href="https://www.w3.org/WAI/WCAG21/Understanding/meaningful-sequence.html">
+                Understanding Success Criterion 1.3.2: Meaningful Sequence
+            </Markup.HyperLink>
+        </Markup.Links>
+        <h4>Sufficient techniques</h4>
```
Comment (peterdur) [inline src/content/test/semantics/letter-spacing.tsx]: Should this be `<h3>` to match how this heading appears in other semantics pages?
```suggestion
        <h3>Sufficient techniques</h3>
```

File: src/content/test/sequence/columns.tsx [hunk 3]
```diff
@@ -0,0 +1,111 @@
...
+            provide for the common defense,
+            ]</div>
+            <div class="column">[promote the
+            general Welfare,
+            and secure
+            the blessings of Liberty
+            to ourselves and our Posterity,
+            do ordain and establish this Constitution
+            for the United States of America.]</div>
+            </body>
```
Comment (peterdur) [inline src/content/test/sequence/columns.tsx]: Should we drop `</body>` and `</html>` here? We're not showing the opening markup for these elements.

File: src/content/test/semantics/letter-spacing.tsx (hunk 2 above)
Comment (ferBonnin) [reply src/content/test/semantics/letter-spacing.tsx]: yes, looks like it was not marked correctly in the doc and when converted it took it as an h4, thanks Peter!

File: src/content/test/sequence/columns.tsx (hunk 3 above)
Comment (ferBonnin) [reply src/content/test/sequence/columns.tsx]: we have the ... on the top, not sure if its bad we don't have the opening markup. I am fine either way

**Pull request merged**